From abe567ff359ced4bc3013265e8c8027166187971 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Tue, 3 Oct 2017 12:37:33 +1000
Subject: [PATCH 2448/4131] amdgpu/dc: don't memset after kzalloc.

We allocate this struct zeroed, so don't need to memset in the
constructor.

Signed-off-by: Dave Airlie <airlied@redhat.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc_surface.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_surface.c b/drivers/gpu/drm/amd/display/dc/core/dc_surface.c
index f170ae9..ff07105 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_surface.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_surface.c
@@ -37,8 +37,6 @@
 static void construct(struct dc_context *ctx, struct dc_plane_state *plane_state)
 {
 	plane_state->ctx = ctx;
-	memset(&plane_state->hdr_static_ctx,
-			0, sizeof(struct dc_hdr_static_metadata));
 }
 
 static void destruct(struct dc_plane_state *plane_state)
-- 
2.7.4

